examples/speaker_recognition/notebooks/Speaker_Recognition_an4.ipynb

```diff
@@ -0,0 +1,1998 @@
+        "eval_tensors = neural_factory.infer(tensors=[embeddings, label_test], checkpoint_dir=\"./myExps/checkpoints/\" + exp_name)\n",
+        "    # inf_loss , inf_emb, inf_logits, inf_label = eval_tensors\n",
+        "inf_emb, inf_label = eval_tensors\n",
+        "whole_embs = []\n",
+        "whole_labels = []\n",
+        "manifest = open(test_dataset, 'r').readlines()\n",
+        "\n",
+        "for line in manifest:\n",
+        "    line = line.strip()\n",
+        "    dic = json.loads(line)\n",
+        "    filename = dic['audio_filepath'].split('/')[-1]\n",
+        "    whole_labels.append(filename)\n",
+        "\n",
+        "for idx in range(len(inf_label)):\n",
+        "    whole_embs.extend(inf_emb[idx].numpy())\n",
+        "\n",
+        "embedding_dir = './myExps/embeddings/'\n",
+        "if not os.path.exists(embedding_dir):\n",
+        "    os.mkdir(embedding_dir)\n",
+        "\n",
+        "filename = os.path.basename(test_dataset).split('.')[0]\n",
+        "name = embedding_dir + filename\n",
+        "\n",
+        "np.save(name + '.npy', np.asarray(whole_embs))\n",
+        "np.save(name + '_labels.npy', np.asarray(whole_labels))\n",
+        "logging.info(\"Saved embedding files to {}\".format(embedding_dir))\n"
+      ],
+      "execution_count": 27,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "[NeMo I 2020-05-11 02:36:07 actions:1533] Restoring JasperEncoder from ./myExps/checkpoints/quartznet3x1_an4/JasperEncoder-STEP-2700.pt\n",
+            "[NeMo I 2020-05-11 02:36:07 actions:1533] Restoring JasperDecoderForSpkrClass from ./myExps/checkpoints/quartznet3x1_an4/JasperDecoderForSpkrClass-STEP-2700.pt\n",
+            "[NeMo I 2020-05-11 02:36:07 actions:759] Evaluating batch 0 out of 5\n",
+            "[NeMo I 2020-05-11 02:36:07 actions:759] Evaluating batch 1 out of 5\n",
+            "[NeMo I 2020-05-11 02:36:07 actions:759] Evaluating batch 2 out of 5\n",
+            "[NeMo I 2020-05-11 02:36:08 actions:759] Evaluating batch 3 out of 5\n",
+            "[NeMo I 2020-05-11 02:36:08 actions:759] Evaluating batch 4 out of 5\n",
+            "[NeMo I 2020-05-11 02:36:08 <ipython-input-27-e3f164d28ce7>:28] Saved embedding files to ./myExps/embeddings/\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "SKKVIb7e6vel",
+        "colab_type": "code",
+        "outputId": "a3fa3703-da6c-4a07-c20c-c83df11a8f25",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 34
+        }
+      },
+      "source": [
+        "ls myExps/embeddings/"
+      ],
+      "execution_count": 28,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "test_labels.npy  test.npy\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
```

Comment:
How come you don't show how to get EER? Is it in the tutorial.rst?

Reply:
an4 doesn't have trial files to show the EER rate. I'll show that in an advanced hi-mia tutorial. Steps are also mentioned in tutorial.rst

Reply:
Can you mention at the end that we will show how to use these embedding in the advanced hi-mia script?

Reply:
Sure